change_log/next/reports_no_scroll_issue.yml

```diff
@@ -0,0 +1 @@
+Bug Fixes: "Fixes the Balance Sheet and Profit and Loss pages would not scroll when opening the full screen dialog"
```

Comment:
This should be more generic to the bug fix in the component, rather than what you're fixing in the app. We shouldn't really mention in app bugs in this open source project :)